lib/Cake/Network/Email/SmtpTransport.php

```diff
@@ -369,7 +369,7 @@ protected function _smtpSend($data, $checkCode = '250') {
 				if ($code[2] === '-') {
 					continue;
 				}
-				return $code[1];
+				return;
```

Comment:
Seems like we should instead fix the doc block. Will do.
Whats up with travis by the way?